superset-frontend/package.json

```diff
@@ -92,7 +92,7 @@
     "@superset-ui/legacy-preset-chart-big-number": "^0.18.19",
     "@superset-ui/legacy-preset-chart-deckgl": "^0.4.13",
     "@superset-ui/legacy-preset-chart-nvd3": "^0.18.19",
-    "@superset-ui/plugin-chart-echarts": "^0.18.19",
+    "@superset-ui/plugin-chart-echarts": "^0.18.22",
```

Comment:
I'm not sure we can do a surgical bump of `@superset-ui/plugin-chart-echarts` from 0.18.19 to 0.18.22, as it will cause two different versions of `@superset-ui/core` to be installed. The only viable option I can think of is to pull in the following `superset-ui` bumps that will end up bringing in that missing cherry:
- #17362
- #17379

Reply:
`superset-ui/plugin-chart-echarts` depends on `superset-ui/core 0.18.22` https://github.com/apache-superset/superset-ui/blob/v0.18.22/plugins/plugin-chart-echarts/package.json